### PR TITLE
added Spec for application env as stage arg

### PR DESCRIPTION
**Pull request by cokieffebah**
# PR Details

## Types of Changes

- [x] Added Unit Testing
- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I have added the appropriate label for this PR
- [ ] If necessary, I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by steven-terrana — APPROVED** [review]

File: docs/antora.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: jte
 title: Jenkins Templating Engine
-version: 2.0-alpha2
+version: 2.0.1
```
Comment (steven-terrana) [inline docs/antora.yml:3]: can we revert this change?

Comment (cokieffebah) [reply docs/antora.yml:3]: what should be the antora version? 2.0.3? or 2.0.4 or something else
